README.md

```diff
@@ -1,14 +1,45 @@
-# Your Project Name
+# Arduino RFID Reader & Writer
```

Comment:
Project might not meet requirement to use at least 3 labs and 3 peripherals. Please add full component list